CMakeLists.txt

```diff
@@ -286,6 +286,10 @@ if (NOT pool-allocators)
 	add_definitions(-DTORRENT_DISABLE_POOL_ALLOCATOR)
 endif()
 
+if (NOT resolve-countries)
+	add_definitions(-DTORRENT_DISABLE_RESOLVE_COUNTRIES)
```

Comment:
This define is not in use anymore. I think your first patch to remove it was correct

Reply:
But it is still better to remove this along with source code cleanup, right?

This could be in a separate PR as finding all ```TORRENT_DISABLE_RESOLVE_COUNTRIES``` references gives:
>Matching lines: 18    Matching files: 10

Reply:
oh, sorry. Yes. I must have git grepped in the wrong directory or something.